extensions/php/languages/php/injections.scm

```diff
@@ -1,3 +1,7 @@
 ((text) @content
  (#set! "language" "html")
  (#set! "combined"))
+
+((comment) @content
+  (#match? @content "^/\\*\\*[^*]")
```

Comment:
Technically, phpdoc comments *must* start with `/**`, so this prevents line comments and block comments that only start with `/*`, or "banner" comments like `/***********/` from being highlighted.